frontend/taipy-gui/src/components/Taipy/Button.tsx

```diff
@@ -61,6 +64,16 @@ const Button = (props: ButtonProps) => {
             <MuiButton id={id} variant="outlined" className={className} onClick={handleClick} disabled={!active}>
                 {typeof value === "string" ? (
                     value
+                ) : (value as Icon).text ? (
+                    <CardHeader
+                        sx={cardSx}
+                        avatar={
+                            <IconAvatar img={value as Icon} className={getSuffixedClassNames(className, "-image")} />
+                        }
+                        title={(value as Icon).text}
+                        disableTypography={true}
+                        className={getSuffixedClassNames(className, "-card")}
```

Comment:
Not too found of this class name... Would '-image-text' make sense to you?
I mean the Card notion is really hidden right?

Reply:
ok